app/views/locations/show.html.haml

```diff
@@ -74,19 +88,30 @@
   = render partial: 'connections/table', locals: {connections: @connected_location.connections}
 
 %br
-.level
-  .level-left
-    = link_to edit_location_path(@location), class: 'button level-item' do
-      %span.icon
-        %i.fa.fa-edit
-      =t('edit')
-  .level-right
-    = link_to @location, method: :delete, data: { confirm: 'Are you sure?' }, class: 'button is-danger' do
-      %span.icon
-        %i.fa.fa-times
-      =t('destroy')
--#= link_to 'Back', locations_path
-    = link_to edit_location_path(@location), class: 'button is-warning level-item' do
-      %span.icon
-        %i.fa.fa-trash
-      =t('destroy')
+-# .level
+-#   .level-left
+-#     = link_to edit_location_path(@location), class: 'button level-item is-primary' do
+-#       %span.icon
+-#         %i.fa.fa-edit
+-#       =t('edit')
+-#     = link_to @location, method: :delete, data: { confirm: 'Are you sure?' }, class: 'button is-danger' do
+-#       %span.icon
+-#         %i.fa.fa-times
+-#       =t('destroy')
+
+
+%section
+  %h1.subtitle
+    %i.fa.fa-map-marker-alt
+    = @location.children.count
+    Child
+    = Location.model_name.human(count: @location.children.count)
+%br
+.columns.is-multiline
+  - @location.children.each do |child|
+    = render 'location_card', location: child
+
+= link_to new_location_path(location: {parent_id: @location.id}), class: 'button is-primary is-outlined' do
+  %span.icon
+    %i.fa.fa-plus
+  %span New Child
```

Comment:
see above ("new Sub location", or "add location within this location" or something without "child").